ui/src/components/Homepage/index.jsx

```diff
@@ -65,9 +65,8 @@ export default class HomePage extends Component {
           </Typography>
         </Hidden>
         <Typography variant="h5">
-          {
-            "A task execution framework that supports Mozilla's continuous integration and release processes."
-          }
+          &quot; A task execution framework that supports Mozilla&apos;s
+          continuous integration and release processes. &quot;
```

Comment:
```suggestion
          continuous integration and release processes.&quot;
```

Reply:
@vishakhanihore yes but why do you need to change symbols into such string?

Reply:
> @vishakhanihore yes but why do you need to change symbols into such string?

Due to the syntax and rule updates.Thanks

Reply:
It looks like the error was
```
WARNING in ./src/components/Homepage/index.jsx
Module Warning (from ./node_modules/eslint-loader/index.js):
error: Curly braces are unnecessary here (react/jsx-curly-brace-presence) at src/components/Homepage/index.jsx:68:11:
  66 |         </Hidden>
  67 |         <Typography variant="h5">
> 68 |           {
     |           ^
  69 |             "A task execution framework that supports Mozilla's continuous integration and release processes."
  70 |           }
  71 |         </Typography>
```

The original form was providing a JS string, and using `{ .. }` so that we could write JS.  I think the reason for this was to allow use of the `'` character without escaping it.  But a change to eslint in this PR is causing that to no longer be accepted.

The `".."` quotes never appeared onscreen - they were just JS syntax.  So I think the right config here is
```diff
-         &quot;A task execution framework that supports Mozilla&apos;s
-         continuous integration and release processes. &quot;
+         A task execution framework that supports Mozilla&apos;s
+         continuous integration and release processes.
```